droopler.install

```diff
@@ -943,3 +944,19 @@ function droopler_update_8105() {
 
 }
 
+function calculateModulesForInstall() {
+  $instalationProfileInfo = \Drupal::service('extension.list.profile')
+    ->getExtensionInfo('droopler');
+  $modules = $instalationProfileInfo['install'];
+  $modulesToInstall = [];
+  foreach ($modules as $module) {
+    if (strpos($module, 'd_') !== FALSE) {
```

Comment:
This will install `d_blog` and `d_product` even if we don't want them, this will also install optional modules if we add some of them in the future.